Raywire/iReporter
# Ch frontend improvements 163222969

**What does this PR do?**
Improves responsiveness no much larger screens by limiting the maximum width and aligning the body to the centre
**Descriptions of the tasks to be completed?**
Add styles to improve responsiveness on larger screens
**How should this be manually tested?**
Go to https://raywire.github.io/iReporter/UI/index.html on a larger screen(over 1650px by 1080px)
Notice that the body is centred
**Pivotal tracker story**
#163222969
